Sources/Kafka/Configuration/KafkaConsumerConfiguration.swift

```diff
@@ -82,6 +82,23 @@ public struct KafkaConsumerConfiguration {
             return .init(consumptionStrategy: .partition(topic: topic, partition: partition, offset: offset))
         }
 
+        /// A consumption strategy based on partition assignment.
+        /// The consumer reads from a specific partition of a topic at a given offset.
+        ///
+        /// - Parameters:
+        ///     - partition: The partition of the topic to consume from.
+        ///     - group: The ID of the consumer group to commit to.
+        ///     - topic: The name of the Kafka topic.
+        ///     - offset: The offset to start consuming from. Defaults to the end of the Kafka partition queue (meaning wait for the next produced message).
+        public static func partition(
```

Comment:
I think we can merge this with the above case. Can we also spell out in the `groupID` doc comment when one would want to provide it when assigning the consumer to a specific partition?